.gitignore

```
@@ -47,3 +47,5 @@
 /site/themes/sogilis/static/css/typeface-montserrat/
 /site/static/admin/netlify-cms.js
 /site/static/admin/netlify-cms.js.map
+
+./.DS_Store
```

```suggestion
.DS_Store
```
Ces dossier peuvent apparaître n'importe où finalement

---

Oupse !

---

Je peux faire la modif et merger cette PR si tu veux.

---

@jibidus done, désolé pour le retard

---

(je pensais que j'avais pushé la correction lol)